.drone.yml

```diff
@@ -17,6 +17,10 @@ steps:
         from_secret: NPM_AUTH
       NPM_EMAIL:
         from_secret: NPM_EMAIL
+      REALM_API_URL:
```

Comment:
I know I brought this up in https://github.com/10gen/devhub-api-service/pull/24, but we never followed up on it. Any reason why you're not using a token / secret to secure this API?

Reply:
So from talking with Grant and researching, we decided that rate limiting was probably the most important thing, and that using a token wasn't super useful given the public nature of this functionality. And I know you mentioned that the [educators page](https://devcenter-frontend.devrel.staging.corp.mongodb.com/developer/academia/educators/) is using tokens, but that form is using the Realm SDK rather than making a call to an API, so I think it's a bit different in how it would function on our site.